config/uwsgi.ini

```diff
@@ -40,5 +40,5 @@ cheaper-rss-limit-hard = 7730941133
 # Configuration to limit the lifespan of a worker to mitigate the
 #  risk of memory leaks in the application or dependencies
 max-requests = 1000
-max-requests-delta = 1
 max-worker-lifetime = 104400
+max-worker-lifetime-delta = 20
```

Comment:
@jasvatos - I'm assuming this is in seconds. Is 20 seconds enough time for an instance to go down and come back up operational before the next instance goes down?

Reply:
Or ... this is the worker process... right not a full instance. 
So --> enough time for a single worker process to be replaced and report healthy before its sibling gets recycled?

Reply:
It is in seconds and doesn't wait for another instance to report healthy. However, the delta is a calculation of `worker_id * delta`. 
With a default of 50 workers, the first worker restarts 20 seconds after the max lifetime is reached. 
The 50th worker will restart 1000 seconds after the max lifetime is reached.